src/Controller/User/Ajax/PasswordStrengthController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Controller\User\Ajax;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use SumoCoders\FrameworkCoreBundle\Security\PasswordStrengthService;
+
+class PasswordStrengthController extends AbstractController
+{
+    public function __invoke(
+        Request $request,
+        PasswordStrengthService $passwordStrengthService
+    ): Response {
+        $password = json_decode($request->getContent(), true)['password'] ?? '';
```

Comment:
Gaat dat geen errors gooien als de request niet volledig is. Of al er geen `password`-key is?

Reply:
Hoorde niet in deze mr